projects/admin/src/app/menu/menu.component.html

```diff
@@ -16,7 +16,7 @@
 -->
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark d-flex">
   <div class="container">
-    <a [routerLink]="['/']" class="navbar-brand pr-2 text-sm">
+    <a [routerLink]="['/']" class="navbar-brand pr-2 text-sm" id="homepage-logo">
```

Comment:
`homepage-logo` is already used in app.component.html. Wouldn't there be conflicts?

Reply:
One appears when you're not authorized, the other one appears when you're logged in. So on my side it seems to be OK.